ios/flutter_reactive_ble.podspec

```diff
@@ -17,11 +14,9 @@ A new flutter plugin project.
   s.dependency 'Flutter'
   s.dependency 'Protobuf', '~> 3.5'
   s.dependency 'SwiftProtobuf', '~> 1.0'
-
-
   s.ios.deployment_target = '10.0'
   s.osx.deployment_target = '10.13'
   s.pod_target_xcconfig = { 'GCC_PREPROCESSOR_DEFINITIONS' => '$(inherited) GPB_USE_PROTOBUF_FRAMEWORK_IMPORTS=1' }
-
+  s.swift_version       = '4.2'
```

Comment:
This is the (technically) important part.

Reply:
does it make sense to consider other deployment targets? like 11 at least?

Reply:
iOS 10.0 or newer is just what we need API-wise.